Test1/src/Problem1.kt

```diff
@@ -0,0 +1,11 @@
+fun main() {
```

Comment:
The implementation is straight forward

The idea was that you get an array of n elements as an input, you don't have to generate the input.